src/main/java/uk/gov/hmcts/darts/authentication/controller/impl/AbstractUserController.java

```diff
@@ -14,18 +14,19 @@
 
 import java.net.URI;
 import java.text.ParseException;
-import java.util.List;
 import java.util.Optional;
 
 @Component
 @RequiredArgsConstructor
 public abstract class AbstractUserController implements AuthenticationController {
 
-    private static final String EMAILS_CLAIM_NAME = "emails";
+    abstract Optional<String> parseEmailAddressFromAccessToken(String accessToken) throws ParseException;
```

Comment:
Should this method be moved to be below the variables?

Reply:
Done